src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
 spring.application.name=GitHubAiAPI
```

Comment:
The property 'spring.application.name' is missing. It is recommended to include this property to specify the name of the application.